test/utils.test.js

```diff
@@ -41,15 +41,15 @@ describe('utils', () => {
 
   describe('isLastLineWhitespaceOnly', () => {
     it('should return true for empty string', () => {
-      expect(isLastLineWhitespaceOnly('')).toEqual(true)
+      expect(isLastLineWhitespaceOnly('')).toEqual(false)
```

Comment:
I don't really get how this fixes the interpolations either. If it does that's great but as I see it `isLastLineWhitespaceOnly` now returns the exact opposite of what I'd expect it to return.

I'd suggest refactoring some more so that the cause of the interpolations breaking is fixed without breaking isLastLineWhitespaceOnly's current behaviour, or at least keeping the behaviour in line with the function name.